src/containers/map/legend/index.tsx

```diff
@@ -185,12 +185,14 @@ const Legend = () => {
                             <DialogTrigger>
                               <Icon
                                 icon={INFO_SVG}
-                                className="mr-1.5 mb-1 h-[18px] w-[18px] fill-black/40"
+                                className="mr-1.5 mb-1.5 h-[17px] w-[17px] fill-black/40"
                               />
                             </DialogTrigger>
-                            <DialogContent className="scroll-y mt-10 h-[90%] rounded-3xl">
-                              <div className="no-scrollbar  overflow-y-auto ">
-                                <Info id={l.id} content={false} />
+                            <DialogContent className="scroll-y mt-10 rounded-3xl" overlay={false}>
+                              <div className="no-scrollbar overflow-y-auto px-3">
+                                <WidgetWrapper key={l.id} title={l.id} id={l.id} info>
```

Comment:
I have problems to fix:
`Type '"planet_medres_visual_monthly"' is not assignable to type 'WidgetSlugType'`